app/models/org.rb

```diff
@@ -171,9 +178,15 @@ def self.default_orgs
               count(users.id) as user_count")
   }
 
-  before_validation :set_default_feedback_email_subject
+  # =============
+  # = Callbacks =
+  # =============
+
+  # this is basically checking the file-store for the dragonfly image each time
+  # before we validate in-case the db/system have moved and the link to the logo files
+  # is therefore different
+  # TODO: evaluate the need for this
   before_validation :check_for_missing_logo_file
-  before_validation :ensure_language
   after_create :create_guidance_group
```

Comment:
I wasn't sure how to remove the callback this case either.
After we make a new organisation, we want to create the default guidance_group for that org.

We could house that logic in the controller where we create organizations instead

Reply:
I think it makes sense to move that outside the model. If we do move this to the controller we will likely need to do the same in the `/app/services/org_selection/hash_to_org.rb`

Reply:
The `hash_to_org` service actually dosen't persist the `Org` object, but rather populates a `Org.new`, and then the persistance is done in the `OrgSelectable`.  

It got me thinking about the purpose of creating the default-guidance-group.  It's unpublished, so really just serves as a way to ensure theres one `GuidanceGroup` with the org's abbreviation and `optional_subset: false` for an org-admin to begin attaching guidance to.

I moved the logic for the method over to the `GuidanceGroup` model, and added a method to the `GuidancesController` to ensure that this default group gets created for any org which is managed and does not currently have a default org